applications/tari_base_node/src/builder.rs

```diff
@@ -20,9 +20,11 @@
 // WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
 // USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-use crate::bootstrap::BaseNodeBootstrapper;
-use log::*;
 use std::sync::Arc;
+
```

Comment:
nit: generally just keep imports together and let rustfmt sort out the order and formatting

Reply:
Yeah I think this was caused by intellij's refactor. Will move it manually next time